source/Weapon.cpp

```diff
@@ -72,6 +72,40 @@ void Weapon::LoadWeapon(const DataNode &node)
 			canCollideAsteroids = false;
 		else if(key == "no minable collisions")
 			canCollideMinables = false;
+		else if(key == "homing")
+		{
+			homing = true;
+			// Convert the old formatting for defining homing for reverse
+			// compatibility.
+			if(child.Size() == 2)
+			{
+				int value = child.Value(1);
+				if(value == 0)
+					homing = false;
+				if(value == 1)
+					blindspot = true;
+				if(value >= 3)
+					throttleControl = true;
+				if(value >= 4)
+					leading = true;
+			}
+			for(const DataNode &grand : child)
+			{
+				for(int j = 0; j < grand.Size(); ++j)
```

Comment:
If at some point we want to expand these attributes even further, this placement flexibility may become an obstacle. My suggestion is to have only the outer loop (each attribute on its own line).

Reply:
Are you suggesting that these attributes should be at the same level of indentation as `homing`?

Reply:
No, I meant
```
homing
	and
	a
	list
	of
	attributes
```
instead of
```
homing
	and a list of attributes
```

Reply:
Isn't that how this PR currently works? It's the same syntax that's used for controlling inaccuracy distribution.

Reply:
Looking at the code, currently both forms are possible. I think we should only allow the first one to have more space if we want to add values to these attributes in the future, for example for more/less accurate "leading" attribute, wider/narrower blindspots etc.